src/libm-tester/CMakeLists.txt

```diff
@@ -55,19 +53,21 @@ macro(test_extension SIMD)
       endif (COMPILER_SUPPORTS_WEAK_ALIASES)
     endif(ENABLE_GNUABI)
 
-    # Build tester2 SIMD
-    string(TOLOWER ${SIMD} SCSIMD)
-    foreach(P dp sp)
-      set(T "tester2${SCSIMD}${P}")
-      add_executable(${T} tester2simd${P}.c testerutil.c)
-      target_compile_options(${T} PRIVATE ${FLAGS_ENABLE_${SIMD}})
-      target_compile_definitions(${T} PRIVATE ENABLE_${SIMD}=1 USEMPFR=1)
-      set_target_properties(${T} PROPERTIES C_STANDARD 99)
-      target_link_libraries(${T} ${TARGET_LIBSLEEF} ${EXTRA_LIBS})
-      add_dependencies(${T} ${TARGET_HEADERS})
-      add_dependencies(${T} ${TARGET_LIBSLEEF})
-    endforeach()
-
+    if(LIB_MPFR AND NOT ${SIMD} STREQUAL NEON32)
```

Comment:
Why do you need to skip NEON32 here? It wasn't skip in the original code.

Reply:
Tester2 does not support NEON32.
In the previous code, it wasn't skipped, but this is a bug.